plutarch-ledger-api/src/Plutarch/LedgerApi/Value.hs

```diff
@@ -275,8 +276,7 @@ pltNonZero ::
   Term s (PValue 'AssocMap.Sorted 'NonZero) ->
   Term s (PValue 'AssocMap.Sorted 'NonZero) ->
   Term s PBool
-pltNonZero t1 t2 =
-  phoistAcyclic (pcheckBinRel #$ phoistAcyclic $ plam (#<)) # t1 # t2
+pltNonZero t1 t2 = pleqNonZero t1 t2 #&& (pnot # (t1 #== t2))
```

Comment:
There should be a smarter way but I don't know how. `pcheckBinRel #<` does not work cause it defaults to `0` in missing case and then there are issues with empty maps, etc.